### ISSUE TITLE
Larger Text and INPUT Fields - Main Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OSHA-Heat-Safety-iOS#6]
The INPUT fields should be bigger and/or at least have more space around them, so people have an easier tap area to hit. People with larger fingers, mobility issues, poor eyesight, etc. will have a hard time activating them. These fields might meet the iPhone standard for tap size, but there is more room on the screen to make them bigger.

WCAG 2.0 - 1.4.4 - Resize Text - Level AA

**Comment by skeeJay** [comment]
We actually went back and forth with OSHA a lot on the size of these values—it has a lot to do with what you consider the primary use case of the app, and thus which element should be most visually prominent. These text fields are larger in the new version than they were in the old one, so it's already a usability improvement and not a regression.

But it's a great suggestion and definitely something that's been debated, and I'm looking forward to having analytics to reopen that design debate with them......if data suggests that users are typing in their own values more than they use their current location, it's a strong argument for making the temperature and humidity more prominent.

**Comment by jfc3-dol** [comment]
Thanks, for making the fields a bit bigger as you mentioned in one of the other issues.

Closing this one.
